packages/block-editor/src/components/list-view/index.js

```diff
@@ -96,6 +104,17 @@ function ListView(
 	const treeGridRef = useMergeRefs( [ elementRef, dropZoneRef, ref ] );
 
 	const isMounted = useRef( false );
+	const { setSelectedTreeId } = useListViewOpenSelectedItem( {
```

Comment:
Just a tiny (and totally optional) name change idea. Since we're dealing with `expand` / `collapse` behaviour, would it be worth renaming this to `useListViewExpandSelectedItem`? Alternately (or and/or) should we add a comment above this line to explain that this hook will ensure that when block selection changes, the selected block (and its parents) are always expanded in the list view?

Mostly I'm just thinking that the logic of the list view is a little challenging to decipher for those new to working with it, so it might help the next person looking at this function. But, we can always look at improving documentation, etc, in follow-ups instead!

Reply:
You're right, thanks for the suggestion. I'll update. 👍